doc/visual-programming/source/widgets/unsupervised/distancematrix.rst

```diff
@@ -0,0 +1,48 @@
+Distance Matrix
+===============
+
+.. figure:: icons/distance-matrix.png
+
+Visualizes distance measures in a distance matrix. 
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Distances**
+
+A distance matrix.
+
+**Outputs**:
+
+-  **Distances**
+
```

Comment:
Missing a description of the output here. Just copy-paste from above and add a description from Table output.